### PR TITLE
fix bug in fake arm and add unit tests

### PR DESCRIPTION
**Pull request by raybjork**
This fixes a bug in the `fake` arm model that returns radians instead of degrees.  Unit tests really should have caught something as trivial as this, so adding those here.

### CODE REVIEW

**Review by ethanlook — APPROVED** [review]
Thank you for the quick fix here!

File: components/arm/fake/fake.go [hunk 1]
```diff
@@ -208,24 +207,19 @@ func (a *Arm) IsMoving(ctx context.Context) (bool, error) {
...
 }
 
-// GoToInputs TODO.
+// GoToInputs moves the fake arm to the given inputs.
 func (a *Arm) GoToInputs(ctx context.Context, inputSteps ...[]referenceframe.Input) error {
 	for _, goal := range inputSteps {
 		a.mu.RLock()
 		positionDegs := a.model.ProtobufFromInput(goal)
 		a.mu.RUnlock()
-		if err := arm.CheckDesiredJointPositions(ctx, a, goal); err != nil {
```
Comment (biotinker) [inline components/arm/fake/fake.go:226]: I think we still want this, no?

Comment (biotinker) [reply components/arm/fake/fake.go:226]: Actually never mind; for a `fake` arm it shouldn't be possible to get slightly OOB like a real arm.

**Review by biotinker — APPROVED** [review]
LGTM

**Pull request merged**